### ISSUE TITLE
.Net Core 6 MySql PowerToolsReadMe.txt not generated corretly

### ISSUE THREAD

**Issue by ezhupa99** [ErikEJ/EFCorePowerTools#1457]
Describe what is not working as expected.

I scaffolded mysql tables into my project, and it did it perfectly and when it finished it showed me some instructions how to register the db service in the Program.cs file which were wrong

### Provide steps to reproduce

Please include a **complete** description or project that I can redo/run to reproduce the issue.

### Provide technical details

- EF Core version in use: (e.g. EF Core 6)

- Is Handlebars used: no/yes
- no

- Is .dacpac used: no/yes

- EF Core Power Tools version: (found in About dialog - blue questionmark icon on context menu)

- Database engine: (SQL Server, Azure SQL, Postgres, Oracle, Firebird, SQLite, MySQL)
- MySQL

- Visual Studio version: (e.g. Visual Studio 2022)
- Visual Studio 2022

This is how it should be

```
var connectionString = builder.Configuration.GetConnectionString("TiCCBoxQA");

builder.Services.AddDbContext<TiCCBoxQAContext>(
    opt =>
    {
        opt.UseMySql(connectionString, ServerVersion.AutoDetect(connectionString));
    });
```

**Comment by ErikEJ** [comment]
What was generated?

And how would Powertools be able to guessing the named connection string??

**Comment by ErikEJ** [comment]
Ah, the MySQL provider does not implement AddMySql - it should - maybe you should use the Pomelo provider instead

**Comment by ezhupa99** [comment]
> Ah, the MySQL provider does not implement AddMySql - it should - maybe you should use the Pomelo provider instead

I have Pomelo installed and the command AddMySql doesn't exist, no intellisense no nothing, it just needs that configuration I provided above.

**Comment by ErikEJ** [comment]
@lauxjpn did you not add this for minimal APIs?

**Comment by ezhupa99** [comment]
What's the use of Minimal API, the registering of service is done the same way

**Comment by ErikEJ** [comment]
Most providers added the AddSqlServer / AddSqlite methods for simplicity

**Comment by ErikEJ** [comment]
Anyway, it is just some hints top get you started...

**Comment by lauxjpn** [comment]
> @lauxjpn did you not add this for minimal APIs?

@ErikEJ Looks like we had missed it. It is now available through our [nightly build feed](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#nightly-builds) and will later be published as part of Pomelo [6.0.3](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/milestone/44).

**Comment by ErikEJ** [comment]
This will solve it self with 6.0.3 and 7.0
